### ISSUE TITLE
Error on edit-config.php page

### ISSUE THREAD

**Issue by Arshad-Abdul** [subjectsplus/SubjectsPlus#1564]
Warning: Undefined array key 1 in C:\xampp\htdocs\sp4\SubjectsPlus-4.5.1\lib\SubjectsPlus\Control\Config.php on line 1411

Warning: Undefined array key "talkback_show_headshot" in C:\xampp\htdocs\sp4\SubjectsPlus-4.5.1\lib\SubjectsPlus\Control\Config.php on line 495

Warning: Undefined array key "talkback_use_email" in C:\xampp\htdocs\sp4\SubjectsPlus-4.5.1\lib\SubjectsPlus\Control\Config.php on line 495

Warning: Undefined array key "talkback_use_recaptcha" in C:\xampp\htdocs\sp4\SubjectsPlus-4.5.1\lib\SubjectsPlus\Control\Config.php on line 495

Warning: Undefined array key "talkback_use_slack" in C:\xampp\htdocs\sp4\SubjectsPlus-4.5.1\lib\SubjectsPlus\Control\Config.php on line 495

Warning: Undefined array key "email_smtp_auth" in C:\xampp\htdocs\sp4\SubjectsPlus-4.5.1\lib\SubjectsPlus\Control\Config.php on line 495

Warning: Undefined array key "problem_report_use" in C:\xampp\htdocs\sp4\SubjectsPlus-4.5.1\lib\SubjectsPlus\Control\Config.php on line 495

Warning: Undefined array key "problem_report_use_recaptcha" in C:\xampp\htdocs\sp4\SubjectsPlus-4.5.1\lib\SubjectsPlus\Control\Config.php on line 495

Warning: Undefined array key "problem_report_use_email" in C:\xampp\htdocs\sp4\SubjectsPlus-4.5.1\lib\SubjectsPlus\Control\Config.php on line 495

Warning: Undefined array key "problem_report_use_slack" in C:\xampp\htdocs\sp4\SubjectsPlus-4.5.1\lib\SubjectsPlus\Control\Config.php on line 495
